### PR TITLE
Drop dashed zone displayed even if dragging far down

### LINKED ISSUE #1347: Groups reorder enhancement - dashed zone at the bottom
This is a nice-to-have.
When dragging a group to the bottom, the dashed zone is displayed and then disappear forever. It should stay visible.

### PR DESCRIPTION
**Pull request by pgiraud**
Fixes #1347.
Pure CSS alternative to #1367.

### CODE REVIEW

File: contribs/gmf/less/desktoplayertree.less [hunk 1]
```diff
@@ -2,12 +2,18 @@
 
 gmf-layertree {
   display: block;
-  margin: @app-margin @half-app-margin @app-margin @app-margin;
+  padding: @app-margin;
+  position: absolute;
+  height: 100%;
```
Comment (sbrunner) [inline contribs/gmf/less/desktoplayertree.less]: 100vh

File: contribs/gmf/less/desktoplayertree.less [hunk 2]
```diff
@@ -2,12 +2,18 @@
...
+  width: 100%;
```
Comment (sbrunner) [inline contribs/gmf/less/desktoplayertree.less]: 100vw

File: contribs/gmf/less/desktoplayertree.less [hunk 3]
```diff
@@ -2,12 +2,18 @@
...
   ul {
     padding-top: @micro-app-margin;
+    height: 100%;
```
Comment (sbrunner) [inline contribs/gmf/less/desktoplayertree.less]: ...

**Comment by sbrunner** [comment]
Sorry my comment are wrong :-/

**Comment by sbrunner** [comment]
Looks good for me :-)

**Comment by pgiraud** [comment]
@olivierSemet I'd like your opinion on this one. I may have missed something.

**Comment by olivierSemet** [comment]
@pgiraud  can you provide an example ?

**Comment by olivierSemet** [comment]
Tested. LGTM

**Comment by olivierSemet** [comment]
Tested. LGTM

**Pull request merged**